Changelog.md

```diff
@@ -6,6 +6,72 @@
 
 ### Ignition Gazebo 6.X.X (202X-XX-XX)
 
+### Ignition Gazebo 6.4.0 (2021-01-13)
```

Comment:
I see `ign-math6` branch on 6.9.3 version. Do we need to sync this or the PR is not related to the latest state of `ign-math`?

Reply:
This PR is targeting Garden, which will use `ign-math7` (i.e. `main`). This 6.4.0 here is Gazebo's changelog

Reply:
aaaaaah! Sorry for the confusion.